tests/application/containers/php/php.Dockerfile

```
@@ -1,9 +1,31 @@
-FROM php:8.5-cli-bookworm
+FROM php:8.5-cli-bookworm AS base
 
 COPY --from=composer/composer:2.9-bin /composer /usr/bin/composer
 
-RUN apt update --quiet --assume-yes \
-    && apt install unzip --quiet --assume-yes --no-install-recommends --no-install-suggests \
-    && pecl install xdebug
+RUN apt-get update --quiet --assume-yes \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)